src/systemdhook.c

```diff
@@ -382,7 +387,18 @@ static int move_mounts(const char *id,
 	snprintf(mount_dir, PATH_MAX, "%s%s", rootfs, path);
 
 	/* Create a temporary directory to move the PATH mounts to */
-	char temp_template[] = "/tmp/ocitmp.XXXXXX";
+	char temp_template[] = "/run/oci/ocitmp.XXXXXX";
+	_cleanup_umount_ char *tempdir = "/run/oci";
```

Comment:
Should we be using something more specific than `/run/oci`?  Now that we have specific needs beyond "is a scratch directory", I'd rather be using `/run/oci-systemd-hook` or something more obviously namespaced to this tool.